packages/xforms-engine/src/error/SelectValueTypeError.ts

```diff
@@ -0,0 +1,22 @@
+import type { SelectDefinition } from '../client/SelectNode.ts';
+import type { UnsupportedSelectValueType } from '../lib/codecs/select/BaseSelectCodec.ts';
+import { XFormsSpecViolationError } from './XFormsSpecViolationError.ts';
+
+/**
+ * @todo It would be good to have a standardized way to use specific types of
+ * errors as a prompt for feedback. There is currently a feedback link presented
+ * by `@getodk/web-forms`, which is conditionally displayed (condition is
+ * evidently controlled by a
+ * {@link https://vuejs.org/api/sfc-script-setup#defineoptions | Vue component option}).
+ *
+ * @see {@link https://github.com/getodk/web-forms/issues/276}
+ */
+export class SelectValueTypeError extends XFormsSpecViolationError {
+	constructor(definition: SelectDefinition<UnsupportedSelectValueType>) {
+		const { valueType } = definition;
+
+		super(
+			`Selects of type ${valueType} are not currently supported. If this functionality would be useful for your form, your feedback is welcome!`
```

Comment:
Suggestion:

This seems like a common type of error for other components. What do you think about using generics in the declaration? Then, in the select definition (SelectNode? or SelectControl?), we could define the error type, extend it, and pass the types through the generic.

By centralizing the possible errors for `select` within select definition (SelectNode? or SelectControl?), it would make them easier to discover and help provide a clearer picture of the entire select functionality.

Reply:
This is a great suggestion. How would you feel about deferring it for now? We can pair on design for it as part of the work on `<odk:rank>`, since it'll have the same type limitations associated with a different control/node type?

Reply:
Yes, that is a good idea!